yarn.lock

```diff
@@ -4703,6 +4703,22 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@metamask/message-signing-snap@npm:^0.2.0":
+  version: 0.2.0
+  resolution: "@metamask/message-signing-snap@npm:0.2.0"
+  dependencies:
+    "@metamask/rpc-errors": "npm:^6.1.0"
+    "@metamask/snaps-jest": "npm:^5.0.0"
```

Comment:
The Snap should not be bringing in `snaps-jest`, that should be marked as a dev dependency.

Reply:
Odd the snap does mark `snaps-jest` as a dev dependency.
https://github.com/consensys-vertical-apps/metamask-message-signing-snap/blob/main/packages/snap/package.json#L46

Unsure how this has been dragged into a normal dep - will investigate.

Reply:
thanks for the heads up on this. Updated the snap dependencies 👍🏾